core/modules/qmeta/QMetaMysql.cc

```diff
@@ -296,9 +297,9 @@ QMetaMysql::registerQuery(QInfo const& qInfo,
         query += _conn.escapeString(itr->second);
         query += "')";
 
-        LOGS(_log, LOG_LVL_DEBUG, "Executing query: " << query);
+        LOGS(_log, LOG_LVL_DEBUG, qIdStr << " Executing query: " << query);
```

Comment:
"Executing query:" should be fine as a first entry in the file. I'll edit the comment in jira. Vaikunth just really needs a reliable entry for the query starting.